packages/strapi-admin/admin/src/components/EventInput/index.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useGlobalContext } from 'strapi-helper-plugin';
+import { FormattedMessage } from 'react-intl';
 
+import formatValue from './utils/formatValue';
```

Comment:
You can create an `index.js` file in utils and export all utils functions to avoid multiple `imports` in the future.

Reply:
I think it's the only function we will have for this small component as utils, but I really like what you said and I just changed it on EditView which have multiple utils.